### PR TITLE
Tag PlotlyJS.jl v0.6.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [sglyon/PlotlyJS.jl](https://github.com/sglyon/PlotlyJS.jl)
Release: [v0.6.0](https://github.com/sglyon/PlotlyJS.jl/releases/tag/v0.6.0)
Travis: [Travis Build Status](https://travis-ci.org/sglyon/PlotlyJS.jl/builds/222115402)
Diff: [vs v0.5.2](https://github.com/sglyon/PlotlyJS.jl/compare/b1ea56717e9ab6c98da8dd68a76649ffaecba612...1382cbcff07cf9d4941adf0fa67fabf97f3da512)
`requires` vs v0.5.2: 
```diff
--- 0.5.2/requires
+++ 0.6.0/requires
@@ -6,3 +6,4 @@
 LaTeXStrings
 DocStringExtensions
 Juno
+DataFrames
```
cc: @sglyon

### CODE REVIEW

File: PlotlyJS/versions/0.6.0/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.5-
```
Comment (tkelman) [inline PlotlyJS/versions/0.6.0/requires]: do you still need to support prereleases of 0.5? if not, drop the trailing dash

**Comment by sglyon** [comment]
Happy to do it, how does that work with attobot releases?

**Comment by tkelman** [comment]
there's a section on the attobot readme, you can replace the tag and release and it will update this pr

**Comment by sglyon** [comment]
wow, that was super easy!

Thanks for the tip -- sorry for not reading the docs before asking

**Pull request merged**